shuup/core/models/_units.py

```diff
@@ -30,6 +31,18 @@
 from ._base import TranslatableShuupModel
 
 
+@lru_cache()
+def get_display_unit(sales_unit):
```

Comment:
when do you reset this cache? shouldn't you do that once the defaul sales unit changes?